### ISSUE TITLE
Inverse learning rate

### ISSUE THREAD

**Issue by AndrewUlmer** [lucidrains/gradnorm-pytorch#10]
In the [paper](https://arxiv.org/pdf/1711.02257) r_{i}(t) is the relative _inverse_ learning rate. [Here](https://github.com/lucidrains/gradnorm-pytorch/blob/ed51e0bb594a2704fc22d9993f350edaff539dc1/gradnorm_pytorch/gradnorm_pytorch.py#L247), it looks like you're computing just the relative learning rate. Feel free to correct me, but shouldn't you have `inv_relative_learning_rate = 1 / relative_learning_rate` somewhere?

**Comment by lucidrains** [comment]
@AndrewUlmer oh shoot, yes I think there is an issue when restoring force is enabled 🤦‍♂️

let me try to fix it on my phone (out with the dog)

**Comment by lucidrains** [comment]
@AndrewUlmer ok, can you see if 0.1.1 makes sense?

**Comment by AndrewUlmer** [comment]
yep, thanks!

**Comment by lucidrains** [comment]
straw hat doggo hehe, you must be a one piece fan

**Comment by AndrewUlmer** [comment]
of course
